docs/introduction/simulators.md

```diff
@@ -1,4 +1,4 @@
-# Simulators
+# Configurable Simulations
```

Comment:
I think this section is more about the actual simulators, so wdyt of "Configure Simulators"?